src/main/java/com/google/sps/servlets/KeywordServlet.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Given a tweet or sentence, the servlet will extract salient terms from those
```

Comment:
[Nit] I think there is a minor grammar issue in this sentence which you can fix by just removing the words "from those"